src/mlpack/core/math/ccov.hpp

```diff
@@ -0,0 +1,242 @@
+/**
+ * @file ccov.hpp
+ * @author Ryan Curtin
+ * @author Conrad Sanderson
+ *
+ * ccov(X) is same as cov(trans(X)) but without the cost of computing trans(X)
```

Comment:
mlpack does use a slightly different coding style as armadillo and since this is no armadillo backport, can you go through the style guidelines and adjust the style.

Reply:
@zoq , Made appropriate changes.